packages/react-query-devtools/package.json

```diff
@@ -14,37 +14,32 @@
   "main": "build/lib/index.js",
   "exports": {
     ".": {
-      "development": {
```

Comment:
generated files `index.js`/`index.mjs` still contain `NODE_ENV` conditions which will be used by the bundler to treeshake it from production